lua/smart-splits/api.lua

```diff
@@ -421,15 +431,8 @@ end
 local function swap_bufs(direction, opts)
   opts = opts or {}
 
-  -- ignore currently focused floating windows by focusing the last accessed window
-  -- if the last accessed window is also floating, do not attempt to swap buffers
-  if utils.is_floating_window() then
-    local prev_win = vim.fn.win_getid(vim.fn.winnr('#'))
-    if utils.is_floating_window(prev_win) then
-      return
-    end
-
-    vim.api.nvim_set_current_win(prev_win)
+  if handle_floating_window() then
+    return
```

Comment:
**note:** Wasn't sure about the "swap buffers" action since I don't use it, it seems this only works with Vim buffers anyway? In that case we don't need to do anything for the `mux` behavior, so I'm not passing a `mux_callback` here.

Reply:
Yeah, that makes sense. Swap buffers is only for vim buffers.